mrblib/mitamae/backend.rb

```diff
@@ -88,20 +90,58 @@ def build_command(commands, user: nil, cwd: nil)
       command
     end
 
-    def run_with_open3(commands, user: nil, cwd: nil)
+    def run_with_open3(commands, user: nil, cwd: nil, log_output_severity: false)
+      spawn_opts = {}
       if user
         # Cannot emulate `:user` option without the shell. Fallback to the slow version.
-        Open3.capture3(@shell, '-c', build_command(commands, cwd: cwd, user: user))
+        commands = [@shell, '-c', build_command(commands, cwd: cwd, user: user)]
       else
         if commands.is_a?(String)
           commands = [@shell, '-c', commands]
         end
 
-        spawn_opts = {}
         if cwd
           spawn_opts[:chdir] = cwd
         end
+      end
 
+      if log_output_severity
```

Comment:
This is what I've tried to tell you: Can we always use the streaming behavior in `run_with_open3` whether `log_output_severity` is false or not? In other words, I want you to stop calling `Open3.capture3` in this method since you implemented a better replacement.

Reply:
:sweat_smile: that makes sense, just pushed a change to do that